shell/assets/translations/en-us.yaml

```diff
@@ -2313,6 +2313,23 @@ growl:
     message: "The creation of this Pod would violate existing restricted policies for the adopted Namespace"
     title: PodSecurity violation
 
+inactivity:
+  title: Session expiring
+  titleExpired: Session expired
+  banner: Your session is about to expire due to inactivity. Any unsaved changes will be lost.
+  bannerExpired: Your session has expired in this tab due to inactivity.
+  content: Click “Resume Session” to keep the session in this tab active or refresh the browser after the session has expired.
+  contentExpired: To return to this page click “Refresh” below or refresh the browser.
+  cta: Resume Session
+  ctaExpired: Refresh
+  settings:
+    title: Inactivity
+    checkboxLabel: Enable inactivity session expiration
+    inputLabel: Inactivity timeout (minutes)
+    information: To change the automatic logout behaviour, edit the authorisation and/or session token timeout values (auth-user-session-ttl-minutes and auth-token-max-ttl-minutes) in the general settings.
```

Comment:
`To change the automatic logout behaviour, edit the authorisation and/or session token timeout values ('auth-user-session-ttl-minutes' and 'auth-token-max-ttl-minutes') in the Settings page.`

The settings could be in `code` elements if they looks nice

Reply:
`in the general settings.` --> `in the Settings page.`

Reply:
` in the General settings page.` --> `in the Settings page.`

If the user looks for a General Settings page they won't find one, but they will find a `Settings` page that's in current context.

![image](https://user-images.githubusercontent.com/18697775/236467538-49e468a9-d9d8-4ff1-9156-8aac82e6ba57.png)

If there really 100% needs to be general in there, it should be `in the general Settings page`.